### PR TITLE
chore: added discord link to all relavant places

### PR DESCRIPTION
**Pull request by hemantwasthere**
Resolves: #1085

## Checklist:

- [x] Performed a self-review of the code
- [ ] Rebased to the last commit of the target branch (or merged it into my branch)
- [x] Linked the issues which this PR resolves
- [ ] Documented the changes in code (API docs will be generated automatically)
- [ ] Updated the tests
- [x] All tests are passing

### CODE REVIEW

**Comment by hemantwasthere** [comment]
Hii @ivpavici ser, 
Please review this PR once!

**Comment by PhilippeR26** [comment]
Should we add the direct link, or the invitation link https://discord.gg/YgsdxEx3 (for people that have not yet subscribed to Starknet Discord)?

**Comment by hemantwasthere** [comment]
Yeah I also think that way, I'll update it

**Comment by hemantwasthere** [comment]
Hii @ivpavici, I have updated the discord link please review it once!

File: README.md [hunk 1]
```diff
@@ -63,6 +63,8 @@ Guides can be found [here](https://www.starknetjs.com/docs/guides/intro) :book:
 
 If you consider to contribute to this project please read [CONTRIBUTING.md](https://github.com/starknet-io/starknet.js/blob/main/CONTRIBUTING.md) first.
 
+You can also ask your query in our [Discord Server](https://discord.com/invite/YgsdxEx3)
```
Comment (ivpavici) [inline README.md:66]:
```suggestion
You can also join our dedicated channel for [Starknet.js](https://discord.com/channels/793094838509764618/927918707613786162) on the [Starknet Discord](https://discord.com/invite/YgsdxEx3)
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -101,3 +101,5 @@ For major changes that markedly transform the existing API or significantly alte
 ## Need help?
 
 If you want to contribute but have any questions, concerns or doubts, feel free to ping maintainers. Ideally create a pull request with `WIP` (Work in progress) in its title and ask questions in the pull request description.
+
+You can also join our [Discord Server](https://discord.com/invite/YgsdxEx3)
```
Comment (ivpavici) [inline CONTRIBUTING.md:105]:
```suggestion
You can also ask your query on our dedicated channel for [Starknet.js](https://discord.com/channels/793094838509764618/927918707613786162) on the [Starknet Discord](https://discord.com/invite/YgsdxEx3)
```

**Review by ivpavici — APPROVED** [review]

**Pull request merged**